style.css

```diff
@@ -42,4 +40,26 @@ p {
     background-color: black;
     height: 45px;
     width: 100%;
+    margin-bottom: 15px;
+}
+
+.credit-card-back .information-wrapper {
+    display: flex;
+    margin-left: 15px;
+    width: 180px;
+    height: 35px;
+    background-color: darkgrey;
 }
+
+.information-wrapper .name {
+    display: flex;
+    font-family: "Brush Script MT";
+    font-size: 26px;
+    margin: 0 0 0 10px;
```

Comment:
if you specify just one value from margin imo it is more clear to write margin-left for example :)